docs/source/markdown/podman-create.1.md

```diff
@@ -22,6 +22,28 @@ Default settings for flags are defined in `containers.conf`. Most settings for
 remote connections use the server's containers.conf, except when documented in
 man pages.
 
+## IMAGE
```

Comment:
Everything in this section needs examples - it's extremely unclear how any of this is used in practice.

Reply:
Also, we need to make it clear that some of these transports (I think not all?) do not pull the image if it is present locally in c/storage.

Reply:
Added examples.